Move to bulk save.

jjbennett opened the pull request:
# Critical Changes

# Changes

# Issues Closed
- Fixes an issue where Bulk Service Delivery rollups caused a row lock error.

# New Metadata

# Deleted Metadata

# Definition of Done
  Refer to [Asteroids DoD document](https://salesforce.quip.com/iq2mAy4i62oM) to see any additional details for the items below
- [ ] Any net new LWC work has Sa11y tests & 50% or above lines JEST test coverage  
- [ ] CRUD/FLS is enforced in Apex
- [ ] Permission sets are updated to account for CRUD|FLS|Tab|Classes
- [ ] Field sets are updated to account for new fields
- [ ] UX approval or UX not necessary
- [ ] Link the pull request and work item by PR comment and Chatter post respectively, e.g. GUS: W-0000000: Work Name (https://github.com/SalesforceFoundation/PMM/pull/303)
- [ ] All **acceptance criteria** have been met
    - [ ] Developer
    - [ ] Code Reviewer
    - [ ] QA
- [ ] PR contains draft release notes
- [ ] QE story level testing completed

jjbennett commented:
[W-14633962](https://gus.lightning.force.com/a07EE00001g7USyYAM)

AndersonTarren commented on force-app/main/default/classes/ServiceDeliveryController.cls:
```diff
@@ -33,6 +33,20 @@ public with sharing class ServiceDeliveryController {
         return true;
     }
 
+    @AuraEnabled
```
I was expecting to see an Apex Test update in this PR based on this change.  Same with the change to the domain class.

AndersonTarren reviewed (changes requested):
Gave it a look this morning, nice job.  I think we'll need to get the test coverage to 100%.

jjbennett commented:
Thanks Tarren for the feedback.
I added the missing tests, should be back to 100%.
I did indeed need data-name, the OR in the querySelector did not work as I hoped.
I cleaned up unused code from the handling of submitting the form.
I fixed an issue where the button was no longer being disabled after Save was clicked.

AndersonTarren reviewed (approved):
Thanks for the review feedback updates, Jenny!
